packages/twenty-front/src/modules/views/utils/mapViewGroupsToRecordGroupDefinitions.ts

```diff
@@ -65,7 +64,7 @@ export const mapViewGroupsToRecordGroupDefinitions = ({
     );
 
     const noValueColumn = {
-      id: viewGroup?.id ?? v4(),
+      id: viewGroup?.id ?? '20202020-c05f-46c9-ae1e-2b3c5c702049',
```

Comment:
This is going to cause conflict issues on the backend side if a view group with this id already exist

Reply:
Good point, as we introduce /backfill noValue groups in backend this will only be use for smooth migrations during 0.40 release. I think it's fine :)